package.json

```diff
@@ -53,18 +84,20 @@
 		}
 	},
 	"ava": {
+		"extensions": [
+			"js",
+			"ts"
+		],
 		"require": [
-			"@babel/register"
-		]
-	},
-	"babel": {
-		"presets": [
-			"@ava/stage-4"
+			"ts-node/register"
 		]
 	},
 	"xo": {
 		"globals": [
 			"atom"
+		],
+		"ignore": [
+			"mocks/**/*"
```

Comment:
Moved this to config so my editor wouldn't automatically format mocks..

Reply:
Turns out this was the cause of the test failures. The test was running `xo` using this config. Removing this `xo` config fixed my test failures..